src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,60 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
 
     public void getStatistic(String fromFileName, String toFileName) {
+        String[] inputData = getDataFromFile(fromFileName);
+        int supply = 0;
+        int buy = 0;
+        for (String line: inputData) {
+            String[] record = line.split(",");
+            if (record[0].equals("supply")) {
+                supply += Integer.parseInt(record[1]);
+            } else {
+                buy += Integer.parseInt(record[1]);
+            }
+        }
+        writeDataToFile(toFileName, supply, buy);
+    }
+
+    private String[] getDataFromFile(String fromFileName) {
+        File fileToRead = new File(fromFileName);
+        StringBuilder builder = new StringBuilder();
+        String[] inputData;
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(fileToRead))) {
+            String value = bufferedReader.readLine();
+            while (value != null) {
```

Comment:
```suggestion
            while (bufferedReader.readLine() != null) {
```

Reply:
This solution will skip lines, because is loop there must be another bufferedReader.readLine(), to append line. And instead of:
supply,30
buy,10
buy,13
supply,17
buy,10
supply,5

will be:
buy,10
supply,17
supply,5